### PR TITLE
add header

### PR DESCRIPTION
**Pull request by Pavlo000**
- [DEMO LINK](https://Pavlo000.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Pavlo000.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,67 @@
...
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
```
Comment (Olexandr6) [inline src/index.html:24]: Don't leave redundant comments in your code; delete this

File: src/index.html [hunk 2]
```diff
@@ -7,8 +7,67 @@
...
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="item">
+            <a
+              href="#Apple"
+              class="link__item is-active"
+            >APPLE</a>
+            <div class="shape"></div>
+          </li>
+          <li class="item item-2">
```
Comment (Olexandr6) [inline src/index.html:43]: Add blank line between `li` elements
```suggestion
          </li>
          
          <li class="item item-2">
```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,71 @@
...
+  font-size: 12px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+  padding: 10px 0;
+  margin-left: 50px;
+
+}
```
Comment (Olexandr6) [inline src/style.css:21]:
```suggestion
.logo {
  display: flex;
  padding: 10px 0;
  margin-left: 50px;
}
```

**Review by Olexandr6 — CHANGES_REQUESTED** [review]
Smt went wrong with tests🤔, but the code is okay.
Check comments👇

File: src/style.css [hunk 4]
```diff
@@ -1 +1,75 @@
 /* Styles go here */
```
Comment (mykola-koval) [inline src/style.css:1]:
```suggestion
```

File: src/index.html [hunk 5]
```diff
@@ -3,12 +3,75 @@
...
+
+          <li class="list__item list_item-5">
+            <a href="#Gadgets" class="list__link">Гаджеты</a>
+          </li>
+
+          <li class="list__item list_item-6">
+            <a href="#Tables" class="list__link">Планшеты
+            </a>
+          </li>
+          <li class="list__item list_item-7">
```
Comment (mykola-koval) [inline src/index.html:66]:
```suggestion
          </li>
          
          <li class="list__item list_item-7">
```

**Review by mykola-koval — APPROVED** [review]

**Pull request closed without merging**